### PR TITLE
Fixed makefile build.

### PR DESCRIPTION
**Pull request by dankan1890**

### CODE REVIEW

File: build/makefile/makefile [hunk 1]
```diff
@@ -31,11 +31,12 @@ SOURCES = $(SRC_NANA) $(SRC_DETAIL) $(SRC_FILESYSTEM) $(SRC_AUDIO) $(SRC_AUDIO_D
 LINKOBJ	= $(SOURCES:.cpp=.o)
 
 $(BIN): $(LINKOBJ)
+	mkdir "../bin"
```
Comment (kirbyfan64) [inline build/makefile/makefile]: This should probably be `mkdir -p "../bin"`.

Comment (nabijaczleweli) [inline build/makefile/makefile]: More likely it should be `mkdir -p $(BIN)`

Comment (kirbyfan64) [inline build/makefile/makefile]: @nabijaczleweli Huh? That would end up creating a directory named `libnana.a`.

Comment (nabijaczleweli) [inline build/makefile/makefile]: Nevermind, I cannot into reading

**Comment by dankan1890** [comment]
mkdir under windows has no options.

**Comment by kirbyfan64** [comment]
What if you put this in the top of the makefile:

```
ifeq ($(OS),Windows_NT)
    mkdir=if not exist $(1) mkdir $(1)
else
    mkdir=mkdir -p $(1)
endif
```

and then change `mkdir "../bin"` to `$(call mkdir,../bin)`.

**Comment by nabijaczleweli** [comment]
or you could, dunno, assume a sane environment

**Pull request closed without merging**